change/@fluentui-react-avatar-b5fcd0d1-345c-4e84-9715-df646c74a3c9.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "update react-icons version to pick up fowardref change",
```

nit: 

```suggestion
  "comment": "chore: Update react-icons version to pick up fowardref change.",
```

ditto all change files